DFC.ServiceTaxonomy.GraphSync/Services/CustomContentDefinitionService.cs

```diff
@@ -131,7 +131,16 @@ public IEnumerable<EditTypeViewModel> LoadTypes()
 
         public void RemoveFieldFromPart(string fieldName, string partName)
         {
-            _orchardCoreContentDefinitionService.RemoveFieldFromPart(fieldName, partName);
+            var typeBeingUpdated = _contentDefinitionManager.GetTypeDefinition(partName);
+
+            if (typeBeingUpdated != null && typeBeingUpdated.Parts.Any(x => x.Name == "GraphSyncPart"))
```

Comment:
Could you change this to `nameof(GraphSyncPart)` to make it more refactor friendly.